fe/fe-core/src/main/java/org/apache/doris/load/Load.java

```diff
@@ -1045,12 +1045,23 @@ private static void initColumns(Table tbl, List<ImportColumnDesc> columnExprs,
             return;
         }
 
+        Set<String> tmpSet = Sets.newHashSet();
```

Comment:
Better add comment to explain this logic.
Give an example is better.

Reply:
ok, I add some comments